### DISCUSSION TITLE
Error with wordpress

### DISCUSSION THREAD

**Discussion by KevinHuerta-T** [google/model-viewer#3468] [Q&A]
Hello community. I come to ask for help or if someone has any idea to solve my mistake.

What happens is that I was using model-viewer to display furniture on my wordpress site but from one moment to another model-viewer is creating a conflict with the links. I don't know if anyone had the same thing happen to them.

If anyone has any idea how to fix the error

**Comment by samaneh-kazemi** [comment]
Hi @KevinHuerta-T, thanks for reaching out. Can you please share a glitch link so we can debug this easier?
